code/modules/reqs/supplypacks.dm

```diff
@@ -32,33 +32,39 @@ OPERATIONS
 	containertype = /obj/structure/closet/crate
 
 /datum/supply_packs/operations/beacons_supply
-	name = "supply beacon"
+	name = "Supply Beacon"
```

Comment:
dont capitalise these

Reply:
Why not? They are names.

Reply:
if you capitalize it at worse it'd come out as "This is a Supply Beacon". This makes absolutely no sense as in English its not a person or the name of a major location or vehicle., it'd confuse people who are fluent and might hurt people who are only semi-fluent in English.
Set it back.

Reply:
because its different from the items, which are not capitalized I believe.

Reply:
The console looks much neater with the items capitalized. This doesn't cause any kind of error like capitalized items do. This only affects the UI and manifests.

Reply:
Then capitalise it in the ui

Reply:
I did that.

Reply:
you did it in the code not tgui

Reply:
How do you want it capitalized then?